php/Ice/greeter/Client.php

```diff
@@ -0,0 +1,21 @@
+<?php
+// Copyright (c) ZeroC, Inc.
+
+require_once 'Ice.php';
+require_once 'Greeter.php';
+
+// Create an Ice communicator to initialize the Ice runtime. This communicator is destroyed automatically at the end of
```

Comment:
I verified the communicator is indeed destroyed even though there is no call to ->destroy